qiime2/core/testing/format.py

```diff
@@ -98,3 +98,19 @@ def single_ints_path_maker(self, num):
             return 'nested/file%d.txt' % num
         else:
             return 'file%d.txt' % num
+
+
+class UnimportableFormat(TextFileFormat):
+    """
+    Useless format which maps to nothing, used for testing the plugin
```

Comment:
Deleting the comment entirely causes `flake8` issues, so I cut the comment down considerably in size and made it more precise.